webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/internal/EventLogAPI.scala

```diff
@@ -201,12 +209,12 @@ class EventLogAPI(
         case eb: EmptyBox =>
           val fail = eb ?~! "Error when fetching event logs"
           JsonResponse(errorFormatter(fail.messageChain), 500)
-      }
+      })
 
     case Get(id :: "details" :: Nil, req) =>
       implicit val prettify = restExtractor.extractBoolean("prettify")(req)(identity).getOrElse(Some(false)).getOrElse(false)
       implicit val action: String = "eventDetails"
-      (for {
+      OldInternalApiAuthz.withReadAdmin((for {
```

Comment:
This could give access to tokens with full write access. Until we hash them I think it should be write here too.